.gitignore

```diff
@@ -9,3 +9,4 @@ paper.run.xml
 *out
 *.ipynb_checkpoints*
 .DS_Store
+tex/_minted-paper
```

Comment:
Have added the pyg and synctex file. If anyone feels strongly about the pdf
file by all means throw it in :)

On Thu, Feb 11, 2016 at 12:15 PM Owen Campbell <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/Axelrod-Python/An-open-reproducible-framework-for-the-study-of-the-iterated-prisoners-dilemma/pull/9#discussion_r52593527>
> :
>
> > @@ -9,3 +9,4 @@ paper.run.xml
> >  *out
> >  *.ipynb_checkpoints*
> >  .DS_Store
> > +tex/_minted-paper
>
> Fine, can't get too worked up about it myself!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Axelrod-Python/An-open-reproducible-framework-for-the-study-of-the-iterated-prisoners-dilemma/pull/9/files#r52593527>
> .
>